.github/workflows/l5kit-lock.yml

```diff
@@ -0,0 +1,42 @@
+name: L5Kit-tests-lock
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  CI-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      working-directory: l5kit
+      run: |
+        pip install --upgrade pip
+        pip install pipenv
+        pipenv sync -d
```

Comment:
is there no need for a `wait` here since all steps run in parallel?

Reply:
As far as I know `jobs` are run in parallel in a workflow, but `steps` and `run` are sequential. I would be very surprised if that was not the case ad we've never experienced any sync issues.
https://resources.github.com/whitepapers/GitHub-Actions-Cheat-sheet/ also seems to confirm it (I haven't found a more official doc that explains it though)

Reply:
Ah wait, I aws looking at buildkite xD never mind!